src/Swarrot/Processor/ExceptionCatcher/ExceptionCatcherProcessor.php

```diff
@@ -6,6 +6,11 @@
 use Swarrot\Processor\ProcessorInterface;
 use Psr\Log\LoggerInterface;
 
+/**
+ * Class ExceptionCatcherProcessor
```

Comment:
I'm trying to follow the http://symfony.com/doc/current/contributing/code/standards.html#documentation